readthedocs/conf.py

```diff
@@ -132,7 +134,11 @@ def __getattr__(cls, name):
 #
 
 html_theme = "sphinx_rtd_theme"
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+html_theme_options = {
+    "collapse_navigation": False,
+    "titles_only": True,
+}
+# html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
```

Comment:
I disabled this line to make the `API documentation` expandable by clicking the + sign. Need to test if there are any side effects.

Reply:
CLMM does not have this line and everything works fine.